doc/source/tutorial/stats/continuous_genhyperbolic.rst

```diff
@@ -0,0 +1,139 @@
+
+.. _continuous-genhyperbolic:
+
+Generalized Hyperbolic Distribution
+===================================
+
+The Generalized Hyperbolic Distribution is defined as the normal variance-mean mixture with Generalized Inverse Gaussian distribution as the mixing distribution.
+The "hyperbolic" characterization refers to the fact that the shape of the log-probability distribution can be described as a hyperbola. Hyperbolic distributions are sometime referred to as semi-fat tailed because their probability density decrease slower than "sub-hyperbolic" distributions (e.g. normal distribution, whose log-probability decreases quadratically), but faster than other "extreme value" distributions (e.g. `pareto` distribution, whose log-probability decreases logarithmically).
+
+Functions
+---------
+
+Different parameterization exists in the literature; SciPy implements the location-scale as in Prause (1999).
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
+        f(x, p, a, b) & = &
+        \frac{(a^2 - b^2)^{p/2}}
+        {\sqrt{2\pi}a^{p-0.5}
+        K_p\Big(\sqrt{a^2 - b^2}\Big)}
+        e^{b(x)} \times \frac{K_{p - 1/2}
```

Comment:
```suggestion
        e^{bx} \times \frac{K_{p - 1/2}
```
Perfectly correct as is, but I read it as a function instead of multiplication. Looks like a holdout from when the location was in the parentheses, too?

Reply:
yes indeed